toplevel/toploop.ml

```diff
@@ -431,7 +432,11 @@ let prepare ppf =
       let objects =
         List.rev (!preload_objects @ !Compenv.first_objfiles)
       in
-      List.for_all (Topeval.load_file false ppf) objects
+      let load_file obj =
+        initialize_toplevel_env' ppf;
```

Comment:
I don't follow why we are reinitialize the environment on each load?
Would it not work to initialize the environment once in `prepare` and remove the call to `initialize_toplevel_env'` in `loop`?

Reply:
Yes, I think so. Fixed, thanks.